test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

```diff
@@ -24,11 +24,4 @@ describe('AuthorisationCaseEvent', () => {
 
     expect(uniqResult).to.eql(nonProd);
   });
-
-  it('should contain a unique case type, case event ID and role (no duplicates) for prod only', () => {
```

Comment:
there is no prod-only file anymore

Reply:
Maybe leave this and the file (empty array)?